wrangler-core/src/main/java/io/cdap/directives/column/Rename.java

```diff
@@ -75,6 +78,11 @@ public List<Row> execute(List<Row> rows, ExecutorContext context) throws Directi
     return rows;
   }
 
+  @Override
+  public Map<String, String> getRenamedFields(Schema inputSchema) {
```

Comment:
I see that you have made changes in rename and settype directives, is there any other directive which would need change to accommodate? 

How are we handling parsing directives?

Reply:
I will revert the changes to those directives, added them here just for testing purposes. Next week I'm planning to raise a PR with changes to all the renaming directives, then the week after that changes to data type changing directives. I already have a list that need to be changed, which includes some of the parsing directives that require specific changes.